src/docs/Components/Buttons/previews/button-icon-only.preview.html

```diff
@@ -1,21 +1,16 @@
 <div class="example">
   <a href="#" class="mc-button mc-button--s mc-button--square">
-    <svg class="mc-button__icon  mc-button__icon--s">
-      <use xlink:href="#iconS" />
-    </svg>
-  </a>
-  <a href="#" class="mc-button mc-button--m mc-button--square">
-    <svg class="mc-button__icon  mc-button__icon--m">
+    <svg class="mc-button__icon">
```

Comment:
first one looks fat and not square to me. (safari)

<img width="273" alt="Capture d’écran 2019-12-10 à 15 05 15" src="https://user-images.githubusercontent.com/14542464/70536347-cd180280-1b5e-11ea-859c-0952f65e8430.png">

Reply:
fixed